round.go

```diff
@@ -123,3 +123,12 @@ func Round(x *big.Rat, prec int, method RoundingMode) *big.Rat {
 	// To force renormalization
 	return x.SetFrac(n, d)
 }
+
+// RoundToInt returns x, rounded as integer. Does not modify x
+func RoundToInt(x *big.Rat, method RoundingMode) *big.Int {
+	c := new(big.Rat).Set(x)
+	Round(c, 0, method)
+
```

Comment:
perhaps should have check IsInt here and panic when not int (better panic than return nonsense)